tests/test_api_shim.py

```diff
@@ -111,133 +112,77 @@ def handle_data(context, data):
 """
 
 
-class TestAPIShim(TestCase):
-    @classmethod
-    def setUpClass(cls):
-        cls.env = TradingEnvironment()
-        cls.tempdir = TempDirectory()
-
-        cls.trading_days = cls.env.days_in_range(
-            start=pd.Timestamp("2016-01-05", tz='UTC'),
-            end=pd.Timestamp("2016-01-28", tz='UTC')
-        )
-
-        equities_data = {}
-        for sid in [1, 2, 3]:
-            equities_data[sid] = {
-                "start_date": cls.trading_days[0],
-                "end_date": cls.env.next_trading_day(cls.trading_days[-1]),
-                "symbol": "ASSET{0}".format(sid),
-            }
+class TestAPIShim(WithDataPortal, ZiplineTestCase):
+    SIM_PARAMS_START = pd.Timestamp("2016-01-05", tz='UTC')
+    SIM_PARAMS_END = pd.Timestamp("2016-01-28", tz='UTC')
+    SIM_PARAMS_DATA_FREQUENCY = 'minute'
 
-        cls.env.write_data(equities_data=equities_data)
+    sids = 1, 2, 3
 
-        cls.asset1 = cls.env.asset_finder.retrieve_asset(1)
-        cls.asset2 = cls.env.asset_finder.retrieve_asset(2)
-        cls.asset3 = cls.env.asset_finder.retrieve_asset(3)
-
-        market_opens = cls.env.open_and_closes.market_open.loc[
-            cls.trading_days]
-        market_closes = cls.env.open_and_closes.market_close.loc[
-            cls.trading_days]
-
-        minute_writer = BcolzMinuteBarWriter(
-            cls.trading_days[0],
-            cls.tempdir.path,
-            market_opens,
-            market_closes,
-            US_EQUITIES_MINUTES_PER_DAY
-        )
-
-        for sid in [1, 2, 3]:
-            write_minute_data_for_asset(
-                cls.env, minute_writer, cls.trading_days[0],
-                cls.trading_days[-1], sid
-            )
-
-        cls.adj_reader = cls.create_adjustments_reader()
-
-        cls.sim_params = SimulationParameters(
-            period_start=cls.trading_days[0],
-            period_end=cls.trading_days[-1],
-            data_frequency="minute",
-            env=cls.env
+    @classmethod
+    def make_equity_info(cls):
```

Comment:
these equities have custom symbols. I do agree that most calls will probably want to be a `make_*_equity_info` call though.